crates/weaver_live_check/src/advice.rs

```diff
@@ -68,33 +69,54 @@ impl Advisor for DeprecatedAdvisor {
     fn advise(
         &mut self,
         sample: SampleRef<'_>,
+        signal: &Sample,
         registry_attribute: Option<Rc<Attribute>>,
         registry_group: Option<Rc<ResolvedGroup>>,
     ) -> Result<Vec<Advice>, Error> {
         match sample {
-            SampleRef::Attribute(_sample_attribute) => {
+            SampleRef::Attribute(sample_attribute) => {
                 let mut advices = Vec::new();
                 if let Some(attribute) = registry_attribute {
                     if let Some(deprecated) = &attribute.deprecated {
                         advices.push(Advice {
                             advice_type: "deprecated".to_owned(),
-                            value: deprecated_to_value(deprecated),
-                            message: deprecated.to_string(),
+                            value: json!({
+                                "attribute_name": sample_attribute.name.clone(),
+                                "deprecation_reason": deprecated_to_reason(deprecated),
+                                "deprecation_note": deprecated.to_string(),
```

Comment:
These keys should be constants so we can ensure consistency. (This comment is to cover all occurrences through this module for value construction).